app/src/lib/ui/Manager/ImportBeneficiaries.svelte

```diff
@@ -338,7 +339,7 @@
 										{:else}
 											<Text
 												value={advisorEmailToPros(proValue ?? '')
-													.map(displayFullName)
+													.map((it) => displayFullName(it))
```

Comment:
Pourquoi avoir eu besoin de rajouter ce `it` ?

Reply:
Bizarrement, si je le met pas ça compile pas. Je pense que c'est à cause des deux paramètres facultatifs, il est confus

Reply:
Oui, `map` passe 3 arguments à son callback : l’item, son index et la liste complète. L’exemple de fail classique est un `map(parseInt)` parce qu’on oublie facilement que `parseInt` prend une base en deuxième argument optionnel.